frontend/src/javascript/codeParser/CodeParserUtils.js

```diff
@@ -113,4 +121,27 @@ export class CodeParserUtils {
     static encloseInTag(content, tag) {
         return `<${tag}>${content}</${tag}>`;
     }
+
+    static getDefaultConverters() {
```

Comment:
Zadanie dla chętnych: w takim formacie jest to mało czytelne, najlepiej by było wrzucić to do jakiegoś pliku .cpp i go odczytać i wkleić tutaj jego zawartość. Tak samo można by było zrobić z tymi makrami, które się wrzuca na początku pliku.